models/models.go

```diff
@@ -67,8 +69,14 @@ func (a UnseeAlertList) Swap(i, j int) {
 	a[i], a[j] = a[j], a[i]
 }
 func (a UnseeAlertList) Less(i, j int) bool {
-	// compare timestamps, if equal compare labels
-	return a[i].StartsAt.After(a[j].StartsAt)
+	// compare timestamps, if equal compare fingerprints to stable sort order
+	if a[i].StartsAt.After(a[j].StartsAt) {
```

Comment:
Another option, since you're returning, is just to return early and remove the `else if`.